src/about-menu/about-menu.html

```diff
@@ -78,24 +78,19 @@
 <body> 
   <div>  
     <img src="./studio-app-icon.png" alt="Studio App Icon" width="64" height="64">
-        <p class="studio-name">Studio by WordPress.com</p>
-        <p class="version">Version <span id="version"></span></p>
+        <p class="studio-name" id="studio-by-wpcom">Studio by WordPress.com</p>
+        <p class="version"><span id="version-text">Version</span> <span id="version"></span></p>
```

Comment:
Let's use only one string here, so RTL languages will work better.

```suggestion
        <p class="version"><span id="version-text">Version x.y.z</span></p>
```

Reply:
Added in https://github.com/Automattic/studio/pull/228/commits/af02dbb33cd562e5aa90cddf51fc6864f72e0ace